src/components/tag.scss

```diff
@@ -0,0 +1,132 @@
+%tag-global {
+  // Public API (allowed to be overridden)
+  display: inline-flex;
+  align-items: center;
+  justify-content: center;
+  padding: var(--op-space-2x-small) var(--op-space-x-small);
+  border-radius: var(--op-radius-pill);
+  appearance: none;
+  background-color: var(--op-color-primary-base);
+  color: var(--op-color-primary-on-base);
+  cursor: pointer;
+  font-size: var(--op-font-x-small);
+  font-weight: var(--op-font-weight-bold);
+  gap: var(--op-space-2x-small);
+  letter-spacing: var(--op-letter-spacing-label);
+  line-height: var(--op-line-height-dense);
+  text-align: center;
+  text-decoration: none;
+  transition: var(--op-transition-input);
+  white-space: nowrap;
+
+  &:hover {
+    background-color: var(--op-color-primary-minus-one);
+    color: var(--op-color-primary-on-minus-one);
+  }
+
+  .material-symbols-outlined {
+    border-radius: var(--op-radius-pill);
+
+    &:hover {
+      background-color: var(--op-color-primary-minus-three);
+      color: var(--op-color-primary-on-minus-three);
+      cursor: pointer;
+    }
+  }
+
+  // Focus State
+  &:focus,
+  &:focus-within,
+  &:focus-visible {
+    outline: none;
```

Comment:
Curious if there is more of a focus state on this. I plan to sync with @scriswell on this as well as the question about intended usage of this soon.